recipes/xrt/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "xrt" %}
+{% set version = "1.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 19c0d85357019d91b324c5ba5ffe7cc6fb31e70f1fa2c9c869d1ee6b63a7338f
+  # TODO: remove the patch after https://github.com/kklmn/xrt/pull/42 is accepted/merged/released.
+  patches:
+    - entry-points.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<36]
+  entry_points:
+    - {{ name }} = {{ name }}.gui.xrtQookStart:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
I don't see anywhere this is using a compiler to install the package...in fact, I think this could be `noarch: python`?

Reply:
You are right it's not building C/C++ extensions. The package uses .cl files, that can probably be handled by the addition of [`pyopencl`](https://github.com/conda-forge/pyopencl-feedstock/blob/master/recipe/meta.yaml) dependency.

Reply:
This is done, the builds are passing now.